specs/037-self-hosted-runners/plan.md

```
@@ -0,0 +1,353 @@
+          cache-to: type=registry,ref=ghcr.io/alkem-io/server:buildcache-release,mode=max
+```
+
+#### 4.2 `build-release-docker-hub.yml` — Delete file
+
+Remove entirely. Consolidated into the new workflow above.
+
+---
+
+## Dependency Graph
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)